### DISCUSSION TITLE
Font size

### DISCUSSION THREAD

**Discussion by Knutsford** [twbs/bootstrap#38868] [Q&A]
I have `<h2 style="color: #FFFFFF;font-size: 1vw;">`

I am trying to get a responsive `<h2>` but that is too big on a PC and too small on a phone so it is unreadable

How do I go about getting `<h2>` to be bootstrapped and responsive so it looks sensible on both I am stumped

**Comment by patkon** [comment]
I don't think you will be happy with inline styles like
`<h2 style="color: #FFFFFF;font-size: 1vw;">`

So actually the headings are already responsive in Bootstrap. If you want to change the sizes, you have to look into the Sass variables:
https://getbootstrap.com/docs/5.3/content/typography/#sass-variables

Especially the H2 is determined in the file _variabless.scss (Line 643):
https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L634

And here you can find the explanation of how the responsive font sizes work in bootstrap
https://getbootstrap.com/docs/5.3/getting-started/rfs/

**Comment by Knutsford** [comment]
I am a bit a bit confused. What files need to be included to get it to work? Is there an example page somewhere as I can’t find one or is it already included in bootstrap 5?

Thanks

**Reply by patkon** [reply]
There are even a lot of [examples](https://getbootstrap.com/docs/5.3/examples/)
For you this should be especially interesting: https://getbootstrap.com/docs/5.3/examples/cheatsheet/

**Comment by Knutsford** [comment]
Thanks. I am trying to get the designer to move over to version 5 now . he was still using 3
